scout/server/blueprints/institutes/forms.py

```diff
@@ -74,8 +81,7 @@ class InstituteForm(FlaskForm):
     )
     institutes = NonValidatingSelectMultipleField("Institutes to share cases with", choices=[])
 
-    loqus_instances = [instance["id"] for instance in loqusdb.loqusdb_settings]
```

Comment:
This code gets called immediately when the app launches, before the code that collects the loqus instances in loqus extension, so it just gets a "default" value.